### PR TITLE
Nightly Build Action & Cleaner README Design

### PR DESCRIPTION
**Pull request by 0PandaDEV**
I added an actions file that compile Scope for windows as .msi Linux as .AppImage and macOS as .app for silicon and intel macs. On top of that, I changed the design of the README to look simpler.

### CODE REVIEW

**Review by Waradu — APPROVED** [review]

**Comment by alii** [comment]
This is great and super useful, thanks. Do all icons work across all platforms? If so then happy to merge. Had a brief look through and it's working on your fork so happy with that part.

**Comment by alii** [comment]
CleanShot 2024-11-16 at 06 39 02@2x
Unable to run on Apple Silicon actually, could this be related to notarization?

**Comment by bvvst** [comment]
ya this is notarization + code signing thing.

testing actions locally:
https://github.com/nektos/act

**Comment by 0PandaDEV** [comment]
Yes, the damaged build is because it's not code signed; however, you can fix that temporarily with the command:
```sh
xattr -cr /Applications/Scope.app
```
But I recommend setting up basic code signing (does **NOT** require a paid Apple developer account) which then shows up as "unverified". If you want me to, I can add that in another pr to this action.

@alii pls merge

**Comment by circularsprojects** [comment]
I have a paid apple developer account I can use to codesign the app if needed

**Comment by 0PandaDEV** [comment]
Oh, that would be perfect I'm currently waiting for this pr to be merged, and then I will create a second one with the code signing in the actions file.

**Comment by alii** [comment]
Thanks, will discuss with the others re/ which developer account we should use. I also have one but it might be that for the time being we just sign without notarizing. Will read through this again and then can merge

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.inputs.ref || github.ref }}
+
+      - name: Install Rust
+        uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: 1.80.0
```
Comment (alii) [inline .github/workflows/build.yml]: Can we set this to `stable`

**Comment by circularsprojects** [comment]
wdymOn 17 Nov 2024, at 1:13 pm, Alistair Smith ***@***.***> wrote:﻿
@alii commented on this pull request.



In .github/workflows/build.yml:
> +        id: get_version
+        run: echo "version=$(grep '^version = ' src/ui/Cargo.toml | cut -d '"' -f2)" >> $GITHUB_OUTPUT
+
+  build-linux:
+    needs: get-version
+    name: Build Linux
+    runs-on: ubuntu-latest
+    steps:
+      - uses: ***@***.***
+        with:
+          ref: ${{ github.event.inputs.ref || github.ref }}
+
+      - name: Install Rust
+        uses: ***@***.***
+        with:
+          toolchain: 1.80.0

Can we set this to stable

**Comment by alii** [comment]
Thanks so much, this is great. Will merge

**Pull request merged**